### PR TITLE
Batch of fixes

### PR DESCRIPTION
**Pull request by gagnieray**
In hope this PR will help you keep this awesome theme working with newer versions of Redmine :crossed_fingers: :rocket:

Fixes issues #222, #233, #242, #245, #246, #253, and few others not already reported :bug: :adhesive_bandage: 
 
Tested on Redmine 4.2.10 and 5.0.5 with the following plugins enabed : 
* [Additionals](https://www.redmine.org/plugins/additionals)
* [Additional Tags](https://www.redmine.org/plugins/additional_tags)
* [Impersonate](https://www.redmine.org/plugins/redmine_impersonate)
* [Dashboard](https://www.redmine.org/plugins/redmine-dashboard)
* [Hearts](https://www.redmine.org/plugins/redmine_hearts)

### CODE REVIEW

**Comment by felixsinger** [comment]
Awesome. Thanks!

File: src/sass/components/_content.scss [hunk 1]
```diff
@@ -343,24 +344,49 @@
...
-    #main {
+    #main,
+    #footer {
       width: $width-sm;
       margin-right: auto;
       margin-left: auto;
     }
+    #main {
+      overflow: hidden;
+      background: $white;
```
Comment (mrliptontea) [inline src/sass/components/_content.scss:361]: Should it be

```suggestion
      background: $body-bg;
```

File: src/sass/components/_plugins.scss [hunk 2]
```diff
@@ -3,6 +3,21 @@
...
+  transform: translateY(-175%);
+  @if $sidebar-position == "right" {
+    right: 15px;
+  }
+}
+
+.flyout-menu .dashboard-system-default {
+  position: absolute;
+  right: 10px;
+  transform: translateY(-190%);
```
Comment (mrliptontea) [inline src/sass/components/_plugins.scss:17]: This (and -175% above) are a bit specific/odd values. Do they work correctly on different window sizes? If yes - can you add a comment and explain why these values?

File: src/sass/components/_plugins.scss [hunk 3]
```diff
@@ -39,6 +54,239 @@
...
+      > ul > li > a,
+      .rdb-multicheck a {
+        padding: 3px 5px;
+      }
+
+      .rdb-checkbox-link.rdb-checkbox-link-enabled {
+        background-image: none;
+
+        &::before {
+          content: "\f00c";
```
Comment (mrliptontea) [inline src/sass/components/_plugins.scss:180]: Use appropriate `$fa-var-`, they're descriptive

```suggestion
          content: $fa-var-check;
```

This applies to all places where icons were added.

File: src/sass/components/_plugins.scss [hunk 4]
```diff
@@ -39,6 +54,239 @@
...
+          display: inline-block;
+          color: $green;
+          font-family: FontAwesome;
```
Comment (mrliptontea) [inline src/sass/components/_plugins.scss:183]: Use the mixin to set the font

```suggestion
          @include fa-icon;
          color: $green;
```

This applies everywhere where FontAwesome was used.

**Review by mrliptontea — COMMENTED** [review]
I am not able to check these changes myself, broadly they look OK. However, issues I see:

- Some of these changes touch basic Redmine layout - were these changes tested in:
   - Different configurations? E.g. fixed layout `true/false`.
   - Earlier Redmine versions? Although I guess it shouldn't matter as much, they're old.
- Few places need to use variables instead of hard-coded values (e.g. icons) for improved readability and consistency.

I also need to make linting GitHub action work.

File: src/sass/components/_content.scss (hunk 1 above)
Comment (gagnieray) [reply src/sass/components/_content.scss]: Fixed!

File: src/sass/components/_plugins.scss (hunk 3 above)
Comment (gagnieray) [reply src/sass/components/_plugins.scss]: Fixed!

File: src/sass/components/_plugins.scss (hunk 4 above)
Comment (gagnieray) [reply src/sass/components/_plugins.scss]: Fixed!

File: src/sass/components/_plugins.scss (hunk 2 above)
Comment (gagnieray) [reply src/sass/components/_plugins.scss]: Yes this works on different windows sizes, but it doesn't when changing some variables.

The icon added by the plugin is outside the `a` tag used to draw the tab. So I solved this by computing existing variables. It should be more explicit.

**Comment by gagnieray** [comment]
> * Some of these changes touch basic Redmine layout - were these changes tested in:
>   * Different configurations? E.g. fixed layout `true/false`.
>   * Earlier Redmine versions? Although I guess it shouldn't matter as much, they're old.

Yes I tried to test theses changes in different configurations : fixed layout + css-grid-layout/$flexbox-layout + sidebar-position.

Regarding earlier Redmine versions, I only tested on the 2 latest releases.

**Review by mrliptontea — APPROVED** [review]
Nice job!

**Pull request merged**